app/uk/gov/hmrc/gatekeeper/views/deploymentApproval/DeploymentApprovalView.scala.html

```diff
@@ -20,66 +20,60 @@
+        @for((api, index) <- unapprovedApis.zipWithIndex) {
+          <tr role="row" class="govuk-table__row">
+            <td class="govuk-table__cell" id="api-servicename-@index">@api.serviceName</td>
+            <td class="govuk-table__cell" id="api-name-@index">@api.name</td>
+            <td class="govuk-table__cell" id="api-environment-@index">@api.env</td>
+            <td class="govuk-table__cell">
+              <a data-review- @api.serviceName class="govuk-link" href="@{routes.DeploymentApprovalController.reviewPage(api.serviceName, api.environment.get.toString)}">
+                Check and approve
+              </a>
+            </td>
+          </tr>
+        }
+      </tbody>
+    </table>
 
-        <table id="application-table"
-               class="dataTable no-footer application-list"
-               width="100%"
-               data-dom="t<'text--center'lip>"
-               data-order='[0,"asc"]'
-               data-page-length="100"
-               data-paginate="true"
-               data-length-change="true"
-               data-length-menu="[10,50,100, 200, 300]"
-               data-responsive="true"
-               data-column-defs='[{"targets": [0,1], "orderable": true, "searchable":true}]'
-               data-language='{"sSearchPlaceholder":"Service ID, Name","sSearch":"<span class=\"full-width shim text--left\"><strong>Search</strong></span>"}'>
-            <thead>
-            <tr role="row">
-                <th tabindex="0" class="sorting_left-aligned">Service ID</th>
-                <th tabindex="0" class="sorting_left-aligned">Name</th>
-                <th tabindex="0" class="sorting_left-aligned">Environment</th>
-                <th class="text--right hard--right" data-orderable="false" tabindex="0">Action</th>
-            </tr>
-            </thead>
-            <tbody>
-            @for((api, index) <- unapprovedApis.zipWithIndex) {
-            <tr role="row">
-                <td id="api-servicename-@index">@api.serviceName</td>
-                <td id="api-name-@index">@api.name</td>
-                <td id="api-environment-@index">@api.env</td>
-                <td class="text--right hard--right">
-                  @if(request.role.isUser) {
-                    <a data-review="@api.serviceName" href="@{routes.DeploymentApprovalController.reviewPage(api.serviceName, api.environment.get.toString)}">Check and approve</a>
-                  }
-                </td>
-            </tr>
-            }
-            </tbody>
-        </table>
-
-        <div class="js-hidden grid-layout grid-layout--stacked" style="margin-left:0">
-            <div class="grid-layout__column--1-3 text--center">
-                @if(unapprovedApis.isEmpty) {"There are no unapproved APIs."}
-            </div>
-        </div>
-
-    </form>
-
-    <div class="form-group>">
-        <a class="back-to-top-link" href="#">Back to Top</a>
+    <div class="govuk-visually-hidden" style="margin-left:0">
```

Comment:
Was the `js-hidden` class linked to some JavaScript?

Reply:
Quite possibly.